cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -69,15 +92,16 @@ export default {
   name: 'Dashboard',
   components: {
     SkeletonLoader,
+    FlyoutMenu,
   },
   data() {
     return {
       cards: [],
       cjson: credentialDisplayData,
       username: '',
       agent: null,
-      error: 'No stored credentials',
-      errorDescription: "Your wallet is empty, there aren't any stored credentials to show.",
+      error: 'No credentials yet',
+      errorDescription: 'Credentials you add will appear here',
```

Comment:
Should be read from i18n resource bundle?

Reply:
@birtony ^^

Reply:
If we have that in designs

Reply:
@sudeshrshetty done